product-guide/manual.adoc

```diff
@@ -205,10 +209,10 @@ The total amount of storage needed depends on the mission use of reports, specif
 |Version Minimum
 
 |podman
-|5.2
+|5.2+
```

Comment:
Have later versions actually been tested? We shouldn't advise what isn't know to work.